src/Avalonia.Base/Media/Fonts/FontCollectionBase.cs

```diff
@@ -12,6 +12,7 @@ namespace Avalonia.Media.Fonts
     public abstract class FontCollectionBase : IFontCollection
     {
         protected readonly ConcurrentDictionary<string, ConcurrentDictionary<FontCollectionKey, IGlyphTypeface?>> _glyphTypefaceCache = new();
+        protected IFontManagerImpl? _fontManager;
```

Comment:
I don't think we should expose such a field publicly (same for `glyphTypefaceCache` above, but that ship has sailed - it should probably be removed in v12).

Instead, `TryCreateSynthethicGlyphTypeface()` could be static and take an extra `IFontManagerImpl` parameter. Each of SystemFontCollection can keep its private existing field.